pyproject.toml

```diff
@@ -43,13 +43,14 @@ requests = "~2.31.0"
 chardet = "^5.2.0"
 # pinning transitive dependencies
 orjson = "~3.9.15"
+idna = "^3.7"
```

Comment:
@CTY-git do we use idna?

Reply:
its a transitive dependency:

![image](https://github.com/patched-codes/patchwork/assets/42710806/e9a914ed-1ff1-4ec1-bf14-40beec301fc0)